<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>571</width>
        <height>36</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Download URL Here</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>511</width>
        <height>36</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Save Location Here</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>140</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="photo.qrc">
        <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>33</height>
       </size>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>571</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>One Video</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>511</width>
          <height>36</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Enter Save Location Here</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>240</y>
          <width>571</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>300</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>511</width>
          <height>36</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Enter Video URL Here</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>190</y>
          <width>101</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>180</y>
          <width>126</width>
          <height>37</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>50</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>33</width>
          <height>26</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>110</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>33</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Playlist</string>
       </attribute>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>260</y>
          <width>571</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>511</width>
          <height>36</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Enter Save Location Here</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>300</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>170</y>
          <width>126</width>
          <height>37</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>180</y>
          <width>101</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>571</width>
          <height>36</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Enter Playlist URL Here</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>101</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Current Video</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>220</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>220</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>220</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>110</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>33</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
